client/src/views/ToolsPage.vue

```diff
@@ -1,12 +1,10 @@
 <template>
   <section>
-    <the-title
-      title="Tools"
-      icon="hammer-screwdriver"
-    />
+    <the-title title="Tools" icon="hammer-screwdriver" />
     <tools-card :items="tools">
       <template #action="{ item }">
-        <dice-roller-dialog :tool="item" />
+        <dice-roller-dialog v-if="item._id === 0" :tool="item" />
```

Comment:
`_id` is the way mongoDB generates ids, I didn't map them, instead, I use the same notation on the frontend. But if we create ids ourselves, we don't need `_`, cause then we can see if id is coming from back or front. But anyway, I'd rather bind to a short name like "dices"  then a number. And this could be done in a method which accepts a name as a parameter, then template would look cleaner